data/discord_messages.yaml

```diff
@@ -1,3 +1,1664 @@
 hello_world:
```

Comment:
Shall we drop this one as we no longer need a placeholder ?

Reply:
This is the only one that show a description